content/shmem_test.tex

```diff
@@ -29,7 +29,13 @@
 \apidescription{
   \FUNC{shmem\_test} tests the numeric comparison of the symmetric object
   pointed to by \VAR{ivar} with the value \VAR{cmp\_value} according to the
-  comparison operator \VAR{cmp}.
+  comparison operator \VAR{cmp}.  The \VAR{ivar} object at the
+  calling \ac{PE} may be updated by an \ac{AMO} performed by a thread located
+  within the calling \ac{PE} or within another \ac{PE}.
+
+  Implementations must ensure that \FUNC{shmem\_test} does not return true before
```

Comment:
My suggestion is : `....the update of the memory indicated by \VAR{ivar} from an \ac{AMO} is fully complete.`
Also, why not remove this-
`Partial updates to the memory must not cause \FUNC{shmem\_test} to return true.`

Reply:
The first part actually is addressing data transfers that the sender might have performed prior to the update to `ivar` and the second is dealing with the `ivar` itself.